app/api/v1/projects.rb

```diff
@@ -51,7 +51,7 @@ def load_projects(archiveds)
       current_time = params[:current_time] || Time.current
       since = params[:since].months.ago
 
-      iteration = Central::Support::IterationService.new(project, current_time: current_time) if project
+      iteration = Central::Support::IterationService.new(project, since: since, current_time: current_time) if project
```

Comment:
Hmm, this can change data delivered to the API. 

We removed this since param intentionally.